tests/test_log_command.rs

```diff
@@ -288,6 +288,50 @@ fn test_log_with_or_without_diff() {
     "###);
 }
 
+#[test]
+fn test_log_divergence() {
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_success(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+
+    std::fs::write(repo_path.join("file"), "foo\n").unwrap();
+    test_env.jj_cmd_success(&repo_path, &["describe", "-m", "description 1"]);
+    let stdout = test_env.jj_cmd_success(
+        &repo_path,
+        &[
+            "log",
+            "-T",
+            "description.first_line() if(divergent, \" !divergence!\")",
```

Comment:
This is fine as is, but in case you didn't know, Rust lets you avoid quoting by adding an 'r' and any number of hashes before and after a string literal, such as `r#"some string with a " in it"#`.

Reply:
That's certainly better, thanks for the reminder!